templates/lxd/install.html

```diff
@@ -0,0 +1,232 @@
+              <div class="p-tabs__item">
+                <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="vm-tab" id="vm"
+                  tabindex="-1">VM</button>
+              </div>
+            </div>
+          </div>
+          <div tabindex="0" role="tabpanel" id="container-tab" aria-labelledby="Container">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}}</pre>
+            </p>
+            <p>Example:</p>
+            <p>
+            <pre>lxc launch ubuntu:22.04 ubuntu-container</pre>
+            </p>
+            <p>Check the <a href="https://images.linuxcontainers.org/">community image server</a> for other Linux distributions.
+            </p>
+          </div>
+          <div tabindex="0" role="tabpanel" id="vm-tab" aria-hidden="true" aria-labelledby="VM">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}} --vm</pre>
+            </p>
+            <p>Example:</p>
+            <p>
+            <pre>lxc launch ubuntu:22.04 ubuntu-container</pre>
+            </p>
+            <p>Check the <a href="https://images.linuxcontainers.org/">community image server</a> for other Linux distributions.
+            </p>
+          </div>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">4.&ensp;Run commands</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>You now have your instance up and running! You're all set to experiment with any
+            commands you need.</p>
+          <p>Command:</p>
+          <p><pre>lxc exec {{"<instance_name> -- <command>"}}</pre></p>
+          <p>Example:</p>
+          <p><pre>lxc exec ubuntu-container -- apt-get update</pre></p>
+          <p>For a list of available commands and options, just run</p>
+          <p><pre>lxc</pre></p>
+        </div>
+      </div>
+      <hr class="p-rule u-hide--small">
+      <p>For other installation options, please <a href="https://documentation.ubuntu.com/lxd/en/latest/installing/">check our documentation</a>.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--25-75 is-split-on-medium">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h2>Firewall issues</h2>
+      </div>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-6 col-medium-4">
+          <p>You might see issues with your firewall blocking network access for your instances, or connectivity issues because you run LXD and Docker on the same host.</p>
+          <p><a href="https://canonical-lxd.readthedocs-hosted.com/en/latest/howto/network_bridge_firewalld/#network-bridge-firewall">Access documentation</a> about how to resolve these issue</p>
```

Comment:
This entire sentence should be the link.

Reply:
We usually don't make entire sentences a link, so that it can create emphasis, I believe this is the usual practice we follow?

Reply:
Nevertheless, I see it has been fixed